src/Scene/LayersConfiguration.js

```diff
@@ -18,6 +18,12 @@ function LayersConfiguration() {
     this.geometryLayers = [];
     this.colorLayers = [];
     this.elevationLayers = [];
+    this.lightingLayers = [];
```

Comment:
The `lightingLayer` allows you to enable or disable all the lights for all other layers

Furthermore each light independently (Moreover each light can be enable)

By adding an `enableLighting` option for each layer and mesh, it is possible to disable the illumination independently for each layer or mesh.